File: API/Domain/Ticket.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Domain
{
    public class Ticket
    {
        public int ticket_id { get; set; }
        public Ticket_Type ticket_type { get; set; }
        public Ticket_Resolution_Type? ticket_resolution_type { get; set; }
        public Ticket_Status ticket_status { get; set; }
        public DateTime created_date { get; set; }
        public string? title { get; set; }
        public string? ticket_description { get; set; }       
        public DateTime? assigned_date { get; set; }        
        public DateTime? resolved_date { get; set; }
        public string? resolution_note { get; set; }
        public virtual ICollection<Ticket_Note>? ticket_notes { get; set; }

    }
}

File: TicketClient/src/app/Models/Ticket.ts
import { Ticket_Type } from "./Ticket_Type";
import { Resolution_Type } from "./Resolution_Type";
import { Ticket_Status } from "./Status";

export interface Ticket {
    ticket_id: number;
    title: string;
    ticket_description: string;
    ticket_type: Ticket_Type;    
    resolution_type: Resolution_Type;    
    ticket_status: Ticket_Status;
    resolution_note: string;
    assigned_date: Date;
    created_date: Date;
    resolved_date: Date;
        
}